.nvmrc

```diff
@@ -0,0 +1 @@
+v20.18.0
```

Comment:
Again, not clear on the use case here?

Reply:
Ah sorry -- this was just so I could use `nvm`/know the version of node that `jco` is expected to work with for development! I can remove if we'd rather not check something like this in!

Reply:
Yeah Node majors happen every 6 months so it's important to get early testing in, and something like this might inhibit noticing changes.

Reply:
What do you think about bumping the version? We can bump it there and in CI as versions change/update etc.

Reply:
It's easy to forget to do it is the problem. That's why CI tests latest by default.

I guess my question is what is the benefit of locking the version? Node.js has strong backwards compat.

Reply:
Yeah this is a reasonable point -- I'm definitely open to taking this out and just having the `latest` be the implied default, with explicit testing for each in CI.

> I guess my question is what is the benefit of locking the version? Node.js has strong backwards compat.

My thought was to make it more consistent for people contributing, *BUT* I agree with your point -- if we're basically expecting that our backwards compat tracks/inherits Node's, it really shouldn't matter!

I'll back this out!